jina/types/document/__init__.py

```diff
@@ -483,7 +451,7 @@ def blob(self, value: Union['np.ndarray', 'jina_pb2.NdArrayProto', 'NdArray']):
         self._update_ndarray('blob', value)
 
     @property
-    def embedding(self) -> 'Union[np.ndarray, scipy.coo_matrix]':
+    def embedding(self) -> 'np.ndarray':
```

Comment:
this type hint should be changed to a TypeVar containing all potential embedding types. planned to add soon